react/components/Table/SwitchablePagination.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import Pagination from '../Pagination'
+
+function SwitchablePagination({ children, enabled, ...paginationProps }) {
+  if (enabled) {
+    return <Pagination {...paginationProps}>{children}</Pagination>
+  }
+  return children
+}
+
+SwitchablePagination.propTypes = {
+  ...Pagination.propTypes,
+  enabled: PropTypes.bool,
```

Comment:
since enabled is not required, you use `defaultProps` as `false`, however undefined is a false value.

Reply:
Done